py/torch_tensorrt/dynamo/runtime/_PythonTorchTensorRTModule.py

```diff
@@ -29,7 +29,7 @@ class PythonTorchTensorRTModule(Module):  # type: ignore[misc]
 
     def __init__(
         self,
-        engine: trt.ICudaEngine,
+        engine: trt.tensorrt.IHostMemory,
```

Comment:
Lets use the bytes interface (may be just changing the type annotation), but we use the same thing for the C++ runtime

Reply:
Thanks. Currently, the `engine` actually saves `serialized engine` since TRT 10 changes API that uses `builder.build_serialized_network`. Do you recommend saving deserialized engines or serialized engines?